docs-source/content/userguide/managing-operators/installation/_index.md

```diff
@@ -34,9 +37,29 @@ $ helm install kubernetes/charts/weblogic-operator \
   --set "javaLoggingLevel=FINE" --wait
 ```
 
+If `weblogic-operator-namespace` exists, then it will be used.  If it does not exist, then Helm will create it.
+
+
+Using Helm 3.x:
+
+```
+$ kubectl create namesapce weblogic-operator-namespace
```

Comment:
1) namesapce --> namespace
2) Sentence above says helm will create the ns if it doesn't exist, so why do you need to create the ns for 3.x?   If the answer is that 3.x behavior has changed, then the sentence above needs to be updated to change behavior.    Otherwise, the create namespace should  be removed.

Reply:
1. Fixed spelling
2. The sentence is for Helm 2.x which automatically creates it.  But Helm 3.x no longer does that